randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3799,7 +3812,7 @@ Extra - asset_id: collision_camera_046
   > Tunnel to EMMI Zone Tower East (2)
       All of the following:
           Morph Ball and After Cataris - Central Unit
-          Simple IBJ or Use Spin Boost
+          Speed Booster or Simple IBJ or Use Spin Boost
```

Comment:
Where do you charge the speed for this? i can only see it needing a door.

Reply:
at the bottom of the room, might need turn around but it can be charged without opening it

Reply:
at the bottom of the room it is possible, it may need a turn around